reference/under_the_hood/indexation.md

```diff
@@ -14,10 +14,51 @@ Crashes caused by memory overconsumption can still happen in two cases:
 
 Multi-threading is unfortunately not possible in machines with only one processor core.
+
+## Speed up the indexation time
+
+If you encounter some performance issues during the indexation we recommend trying the following points:
+
+1. Make sure you are using the latest [stable version of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases). Our latest releases often include performance optimizations that can significantly increase indexation speed.
+
+2. Indexation is a memory-intensive and multi-threaded operation. This means that **the more memory and processor cores available, the faster MeiliSearch will index new documents**. In general, using a machine with many processor cores will be more effective than increasing RAM.
+
+3. **Bigger HTTP payloads are processed more quickly than smaller payloads**. For example, adding the same 100,000 documents in two batches of 50,000 documents will be quicker than in four batches of 25,000 documents. By default, MeiliSearch sets the maximum payload size to 100MB, but [you can change this value if necessary](/reference/features/configuration.md#payload-limit-size). That said, **the bigger the payload, the higher the memory consumption**. An instance may crash if it requires more RAM than is currently available in a machine.
+
+4. **MeiliSearch should not be your main database**. By this we mean you should only index documents you want to retrieve when searching. The more documents you add, the longer will indexation and search take.
+
+5. By default, all the fields of your documents are considered as "searchable". We strongly recommend changing this by [setting the `searchablaAttributes`](https://docs.meilisearch.com/reference/api/searchable_attributes.html#update-searchable-attributes) with the exhaustive list of fields you want to search in. The fewer fields MeiliSearch needs to index, the faster is the indexation process.
+For example, if indexing the `id` and `genre` fields has little impact on the quality of your search, adding only `title` and `author` to `searchableAttributes` will increase indexation speed:
+
+```json
+[
+  { "id": 1, "title": "Pride and Prejudice", "author": "Jane Austin", "genre": "romance" },
+  { "id": 2, "title": "Le Petit Prince", "author": "Antoine de Saint-Exupéry", "genre": "adventure" }
+]
+```
+
+6. When creating a new index, first [configure its settings](/reference/features/settings.md) and only then add your documents. Following this order will significantly reduce indexation time
+
+7. Since indexation speed is tightly connected to the size of your payload, using lightweight dataset formats such as CSV and NDJSON can lead to increased performance
```

Comment:
```suggestion
- Since indexation speed is tightly connected to the size of your payload, using lightweight dataset formats such as CSV and NDJSON can lead to increased performance
```